apps/dotcms-ui/src/app/portlets/dot-apps/dot-apps-import-export-dialog/dot-apps-import-export-dialog.component.html

```diff
@@ -6,6 +6,7 @@
     [actions]="dialogActions"
```

Comment:
<img width="819" alt="Screen Shot 2021-09-30 at 2 57 41 PM" src="https://user-images.githubusercontent.com/751424/135529722-284087be-f6c7-4d26-abf2-081cdd98f8da.png">

Reply:
*labels.

Reply:
no, primeng `FileUpload` does not work as expected.

labels added!